.github/workflows/ci-docs.yaml

```diff
@@ -58,6 +58,19 @@ jobs:
     runs-on: ${{ matrix.os }} 
     steps:
       - run: echo "Skipping CI for docs & contrib files"
+  windows-e2e-tests:
+      strategy:
+        matrix:
+          os: 
+            [
+              [self-hosted, macos, amd64, 13, test], 
```

Comment:
These must be the same architectural types as the actual ci. I think from there, it looks something like this:
```yaml
      matrix:
        os:
          [
            [self-hosted, windows, amd64, test],
          ]
```

Reply:
Fixed in latest rev